tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -109,317 +49,306 @@ describe('Target aggregates', () => {
-    });
+    describe('Online user with one place associated', () => {
+      beforeEach(async () => {
+        await loginPage.login(onlineUser);
+        await commonPage.waitForPageLoaded();
+      });
 
-    it('should display no data when no targets are uploaded', async () => {
-      await loginPage.login({ username: user.username, password: user.password });
-      await commonPage.waitForPageLoaded();
-      const targetsConfig = [
-        { id: 'not_aggregate', type: 'count', title: generateTitle('my task') },
-        { id: 'count_no_goal', type: 'count', title: generateTitle('count no goal'), aggregate: true },
-        { id: 'count_with_goal', type: 'count', title: generateTitle('count with goal'), goal: 20, aggregate: true },
-        { id: 'also_not_aggregate', type: 'count', title: generateTitle('my task') },
-        { id: 'percent_no_goal', type: 'percent', title: generateTitle('percent no goal'), aggregate: true },
-        {
-          id: 'percent_with_goal',
-          type: 'percent',
-          title: generateTitle('percent with goal'),
-          aggregate: true,
-          goal: 80
-        },
-        { id: 'also_also_not_aggregate', type: 'count', title: generateTitle('my task'), aggregate: false },
-      ];
-
-      await updateSettings(targetsConfig, user);
+      it('should display no data when no targets are uploaded', async () => {
+        await helperFunctions.updateSettings(
+          targetAggregatesConfig.TARGETS_CONFIG_WITH_AND_WITHOUT_AGGREGATES, onlineUser
+        );
 
-      await commonPage.goToAnalytics();
-      await targetAggregatesPage.goToTargetAggregates(true);
+        await commonPage.goToAnalytics();
+        await targetAggregatesPage.goToTargetAggregates(true);
 
-      const expectedTargets = [
-        { id: 'count_no_goal', title: 'count no goal', counter: '0', progressBar: false, goal: false },
-        { id: 'count_with_goal', title: 'count with goal', counter: '0 of 5', progressBar: true, goal: 20 },
-        { id: 'percent_no_goal', title: 'percent no goal', counter: '0%', progressBar: true, goal: false },
-        { id: 'percent_with_goal', title: 'percent with goal', counter: '0 of 5', progressBar: true, goal: '80%' },
-      ];
-      const expectedContacts = docs
-        .filter(doc => doc.type === 'person' && doc.parent.parent._id === parentPlace._id)
-        .map(contact => ({ _id: contact._id, name: contact.name, counter: 'No data', progress: 0 }));
-
-      await expectTargets(expectedTargets);
-
-      for (const target of expectedTargets) {
-        await targetAggregatesPage.openTargetDetails(target.id);
-        await targetAggregatesPage.expectTargetDetails(target);
-        await expectContacts(expectedContacts, target);
-      }
-    });
+        const expectedContacts = helperFunctions.getDocsByPlace(contactDocs, districtHospital1._id)
+          .map(contact => ({ _id: contact._id, name: contact.name, counter: 'No data', progress: 0 }));
```

Comment:
1 function per line when chaining functions.
```suggestion
        const expectedContacts = helperFunctions
          .getDocsByPlace(contactDocs, districtHospital1._id)
          .map(contact => ({ _id: contact._id, name: contact.name, counter: 'No data', progress: 0 }));
```